notebooks/259-softvc-voice-conversion/README.md

```diff
@@ -0,0 +1,21 @@
+# SoftVC VITS Singing Voice Conversion and OpenVINO™
```

Comment:
does it work in Colab or Binder? If it does please add badges

Reply:
Only in Colab it works. Binder creates a VM with only 2GB RAM, that is not enough.